OOStepByStep/Person.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace OOStepByStep
+{
+    using System;
+
+    public class Person
+    {
+        protected string name;
```

Comment:
Use {get; set} method is better than protected from security.

Reply:
got it, thanks~